src/render/Renderer.cpp

```diff
@@ -228,6 +228,8 @@ bool CHyprRenderer::shouldRenderWindow(CWindow* pWindow, CMonitor* pMonitor, CWo
         return true;
 
     if (pWindow->m_vRealPosition.isBeingAnimated()) {
+        if (PWINDOWWORKSPACE && PWINDOWWORKSPACE->m_vRenderOffset.isBeingAnimated())
```

Comment:
not that simple, if you are for example doing swapcurrentworkspaces, it should be visible IMO.

Reply:
It is visible though. Tested `swapactiveworkspaces` and `movecurrentworkspacetomonitor`. The offset is not in effect during these operations.

Reply:
my bad